ecosystem/sep-0006.md

```diff
@@ -1044,13 +1078,25 @@ Example response:
       "id": "82fhs729f63dh0v4",
       "kind": "withdrawal",
       "status": "completed",
-      "amount_in": "500",
+      "amount_in": "510",
       "amount_out": "495",
-      "amount_fee": "3",
+      "amount_fee": "5",
       "started_at": "2017-03-20T17:00:02Z",
       "completed_at": "2017-03-20T17:09:58Z",
       "stellar_transaction_id": "17a670bc424ff5ce3b386dbfaae9990b66a2a37b4fbe51547e8794962a3f9e6a",
-      "external_transaction_id": "2dd16cb409513026fbe7defc0c6f826c2d2c65c3da993f747d09bf7dafd31093"
+      "external_transaction_id": "2dd16cb409513026fbe7defc0c6f826c2d2c65c3da993f747d09bf7dafd31093",
+      "refunds": {
+        "amount_refunded": "10",
+        "amount_fee": "0",
```

Comment:
Could we add another example with fee values > 0 to help with the understanding?

Reply:
I just updated the example to use a non-zero refund fee.